frontend/src/flashcard.tsx

```diff
@@ -28,6 +34,86 @@ interface FlashcardProps {
   isPaywalled: boolean;
 }
 
+type TiledTextProps = {
+  text: string;
+} & Pick<PaperProps, "bg" | "c" | "h" | "w" | "withBorder" | "shadow"> &
+  Pick<TextProps, "size" | "fw" | "ff">;
+
+const TiledText: React.FC<TiledTextProps> = ({
+  text,
+  bg,
+  c,
+  h,
+  w,
+  fw,
+  ff,
+  size,
+  withBorder,
+  shadow,
+}) => {
+  return (
+    <Group gap="xs" wrap="wrap">
+      {text.split("").map((char, index) => (
+        <Paper
+          h={h}
+          w={w}
+          key={index}
+          shadow={shadow}
+          bg={bg}
+          withBorder={withBorder}
+        >
+          <Center w="100%" h="100%">
+            <Text c={c} size={size} fw={fw} ff={ff} ta="center">
+              {char}
+            </Text>
+          </Center>
+        </Paper>
+      ))}
+    </Group>
+  );
+};
+
+type QuestionDisplayProps = {
+  displayQuestion: string;
+  isDark: boolean;
+  fontStyle: WordVaultFontStyle;
+  theme: MantineTheme;
+};
+
+const QuestionDisplay: React.FC<QuestionDisplayProps> = ({
+  displayQuestion,
+  isDark,
+  fontStyle,
+  theme,
+}) => {
+  switch (fontStyle) {
+    case WordVaultFontStyle.Tiles: {
+      return (
+        <TiledText
+          size="xxl"
+          h={rem(40)}
+          w={rem(40)}
+          fw={700}
+          withBorder={!isDark}
+          shadow={isDark ? "xs" : undefined}
+          bg={isDark ? theme.colors.gray[8] : theme.colors.gray[4]}
+          c={isDark ? theme.colors.gray[0] : undefined}
+          text={displayQuestion}
+        />
+      );
+    }
+    case WordVaultFontStyle.Monospace:
+    case WordVaultFontStyle.SansSerif:
+    default: {
+      return (
+        <Text size="xxl" fw={700} ta="center" ff={fontStyle}>
```

Comment:
just flagging that I've also bumped the default text size to `xxl`, feel free to revert

Reply:
xxl seems fine; i think the question is a little small myself.